src/main/java/teammates/ui/webapi/RequestTraceFilter.java

```diff
@@ -77,11 +79,16 @@ public void doFilter(ServletRequest req, ServletResponse resp, FilterChain chain
         RequestTracer.init(traceId, spanId, timeoutInSeconds);
 
         Map<String, Object> requestDetails = new HashMap<>();
+        Map<String, Serializable> requestParams = HttpRequestHelper.getRequestParameters(request);
         requestDetails.put("requestMethod", request.getMethod());
         requestDetails.put("requestUrl", request.getRequestURI());
         requestDetails.put("userAgent", request.getHeader("User-Agent"));
-        requestDetails.put("requestParams", HttpRequestHelper.getRequestParameters(request));
+        requestDetails.put("requestParams", requestParams);
         requestDetails.put("requestHeaders", HttpRequestHelper.getRequestHeaders(request));
+        Map<String, Object> userInfo = new HashMap<>();
```

Comment:
This cannot be changed to `Map<String, String>` as the values of `requestParams` are `Serializable`. I think it can be changed to `Serializable` instead.